test/e2e/alphaupdate/update_test.go

```diff
@@ -295,3 +350,40 @@ func validateCustomCodePreservation(projectDir string) {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(string(content)).To(ContainSubstring(controllerImplementation))
 }
+
+func validateConflictMarkers(projectDir string) {
+	By("validating conflict markers are present")
+	filesToCheck := []string{
+		filepath.Join(projectDir, "api", "v1", "testoperator_types.go"),
+		filepath.Join(projectDir, "internal", "controller", "testoperator_controller.go"),
+	}
+
+	conflictMarkersFound := false
+	for _, file := range filesToCheck {
+		content, err := os.ReadFile(file)
+		if err != nil {
+			continue
+		}
+		fileContent := string(content)
+		if strings.Contains(fileContent, "<<<<<<<") && strings.Contains(fileContent, "=======") && strings.Contains(fileContent, ">>>>>>>") {
+			conflictMarkersFound = true
```

Comment:
The problem here is the size of line 
broke it in 2 lines ( just give a enter ) 

Why?
Big lines make it harder to read the code. 
That is why it is a good practice to avoid using the scroll to read everything. 
defauly 120 characters